s3repo/view.py

```diff
@@ -0,0 +1,85 @@
+"""View for working with the repositories on S3."""
+
+import os
+
+from flask.views import View
+from flask import jsonify
+from flask import render_template
+from flask import Response
+
+
+class S3View(View):
+    """View for working with S3 according to the REST model."""
+
+    def __init__(self, model):
+        self.model = model
+
+    @staticmethod
+    def response_message(message, status):
+        """Generate response with a message."""
+        response = jsonify({'message': message})
+        response.status_code = status
+        return response
+
+    @staticmethod
+    def get_directory(abs_path, rel_path, bucket_name, items):
+        """Display directory content as a HTML page."""
+        displayed_path = os.path.normpath('/'.join(['', bucket_name, abs_path]))
```

Comment:
Displayed path must not contain bucket name and base_path.

Reply:
Fixed.